etc/kayobe/environments/aufn-ceph/neutron.yml

```diff
@@ -0,0 +1,15 @@
+---
+kolla_neutron_ml2_network_vlan_ranges:
+  - physical_network: "physnet1"
+
+kolla_neutron_ml2_type_drivers:
+  - flat
+  - vlan
+  - vxlan
```

Comment:
The playbooks within the open stack-network-config repo try to create VXLAN networks, so unless we modify that config for this use case then no we can't remove them

Reply:
If we're using OVN as the networking driver then we should create GENEVE tenant networks rather than VXLANs. Is it possible to omit the network type in openstack-config and let neutron choose for you?